File: Material/Lab06_20231523/Lab06_20231523/Lab06-Estudiantes/java/TransitSoft/TransitSoftModelo/src/main/java/pe/edu/pucp/transitsoft/modelo/Camara.java
package pe.edu.pucp.transitsoft.modelo;

public class Camara {
    private int id;
    private String modelo;
    private String codigoSerie;
    private long latitud;
    private long longitud;

    public Camara() {
        this.id = -1;
        this.modelo = null;
        this.codigoSerie = null;
    }
    
    

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public String getCodigoSerie() {
        return codigoSerie;
    }

    public void setCodigoSerie(String codigoSerie) {
        this.codigoSerie = codigoSerie;
    }

    public long getLatitud() {
        return latitud;
    }

    public void setLatitud(long latitud) {
        this.latitud = latitud;
    }

    public long getLongitud() {
        return longitud;
    }

    public void setLongitud(long longitud) {
        this.longitud = longitud;
    }

    public void setString(String string) {
        throw new UnsupportedOperationException("Not supported yet."); // Generated from nbfs://nbhost/SystemFileSystem/Templates/Classes/Code/GeneratedMethodBody
    }
}

File: Material/TransitSoft/TransitSoftModel/Camara.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TransitSoft.Model
{
    public class Camara
    {
        private int? id;
        private string modelo;
        private string codigoSerie;
        private long? latitud;
        private long? longitud;

        public int? Id { get => id; set => id = value; }
        public string Modelo { get => modelo; set => modelo = value; }
        public string CodigoSerie { get => codigoSerie; set => codigoSerie = value; }
        public long? Latitud { get => latitud; set => latitud = value; }
        public long? Longitud { get => longitud; set => longitud = value; }
    }
}
